test/strings/types.jl

```diff
@@ -188,6 +188,25 @@ let s = "lorem ipsum", sdict = Dict(
     end
 end
 
+# proper nextind/prevind/thisind for SubString{String}
+let s = "∀∃∀", ss = SubString(s, 4, 4), s2 = s[4:4]
```

Comment:
Is this meant to be just one character for the test only? Maybe test more substrings?